modules/packages/msiexec.bat

```diff
@@ -96,8 +96,12 @@ rem Install this file if it exists
     goto :EOF
   )
 
-  %MSIEXEC% /quiet /passive /qn /norestart /i %1
-  rem TODO options, error checking
+  REM TODO: ENT-6824 save this logfile based on msi filename
+  set logfile="\cfengine_package_install.log"

```

Comment:
Where on the file system is this log file?
I think we /must/ have something to prevent it from growing without bound. We have log file control in the MPF here: https://github.com/cfengine/masterfiles/blob/master/cfe_internal/core/log_rotation.cf

Reply:
it is only one file. in the root directory. It cannot grow without bound I think.

Reply:
I might just include ENT-6824 (unique log file names) and some retention policy then in this PR?